liquibase-integration-tests/src/test/java/liquibase/dbtest/oracle/OracleIntegrationTest.java

```diff
@@ -225,7 +225,7 @@ public void testChangeLogGenerationForTableWithGeneratedColumn() throws Exceptio
         String textToTest = "GENERATED ALWAYS AS (QTY * PRICE)";
 
         Scope.getCurrentScope().getSingleton(ExecutorService.class).getExecutor("jdbc", getDatabase()).execute(new RawSqlStatement(
-            String.format("CREATE TABLE GENERATED_COLUMN_TEST(QTY INT, PRICE INT, TOTALVALUE INT %s);", textToTest)));
```

Comment:
Oops.

Reply:
Almost there :)
The assertion failed as it is not able to find our sql fragment. The generated SQL below is:
```
<column defaultValueComputed="GENERATED ALWAYS AS (&quot;QTY&quot;*&quot;PRICE&quot;)" name="TOTALVALUE" type="NUMBER(*, 0)"/>
```
So in my next commit I'm removing `"` from virtColumnDef variable.. and seems it works. I wonder if this is the best solution @LonwoLonwo  @MalloD12 , but at least it is working now :)